phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/Repository.java

```diff
@@ -38,6 +38,7 @@ default void fetchInstallableApplications(Consumer<RepositoryDTO> callback, Cons
         try {
             callback.accept(fetchInstallableApplications());
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Please use a logger here.
In addition we shouldn't catch `Exception` here but a more specific subclass